js/searchColumns_event.js

```diff
@@ -0,0 +1,8 @@
+CRM.$(function($) {
+  // Foreach data that has inventoryfield
+  $.each(CRM.vars.eventRows, function(key, value) {
```

Comment:
please use more descriptive variable names related to `eventRows`, e.g. `function(rowKey, row) {`